repositories/src/main/scala/cromwell/pipeline/datastorage/dto/ProjectResponse.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.pipeline.datastorage.dto
+
+import play.api.libs.json.{Json, OFormat}
+
+final case class ProjectResponse(
+  projectId: ProjectId,
+  name: String,
+  active: Boolean
+)
+
+object ProjectResponse {
+  implicit val findProjectResponse: OFormat[ProjectResponse] = Json.format[ProjectResponse]
+}
```

Comment:
Why it's located in the separate class and not in `Project.scala`?

Reply:
Because any way all implicit object's for formatting into json-format will be relocated.

Reply:
Look's like it's useless object. It will be removed.

Reply:
done